src/wfdirrd.c

```diff
@@ -1220,7 +1220,7 @@ DWORD DecodeReparsePoint(LPCWSTR szMyFile, LPCWSTR szChild, LPWSTR szDest, DWORD
 	StripFilespec(szFullPath);
 
 	if (szChild != NULL)
-		AppendToPath(szFullPath, szChild);

```

Comment:
The second parameter is now LPCTSTR.
https://github.com/Microsoft/winfile/commit/d0493ea4e94eb8b248f9866748d23bb7e332eb47